src/main/java/uk/gov/hmcts/reform/professionalapi/util/FeatureConditionEvaluation.java

```diff
@@ -57,17 +58,20 @@ public boolean preHandle(HttpServletRequest request,
         return flagStatus;
     }
 
-    public String getServiceName(String flagName) {
+    public String getServiceName() {
 
         ServletRequestAttributes servletRequestAttributes =
             ((ServletRequestAttributes) RequestContextHolder.getRequestAttributes());
 
         if (nonNull(servletRequestAttributes)) {
             HttpServletRequest request = servletRequestAttributes.getRequest();
-            return JWT.decode(removeBearerFromToken(request.getHeader(SERVICE_AUTHORIZATION))).getSubject();
+
+            if (StringUtils.isNotEmpty(request.getHeader(SERVICE_AUTHORIZATION))) {
+                return JWT.decode(removeBearerFromToken(request.getHeader(SERVICE_AUTHORIZATION))).getSubject();
+            }
         }
 
-        throw new ForbiddenException(flagName.concat(SPACE).concat(FORBIDDEN_EXCEPTION_LD));
+        return null;
```

Comment:
i think it my bad but we have to use optional to avoid null returns and returns optional.empty